scripts/run_translation_templates.py

```diff
@@ -0,0 +1,144 @@
+            '-o',
+            output_filepath,
+            '.',
+        ]
+    )
+
+
+def check_static_template(source_dir, target_dir):
+    source_contents = get_template_content(
+        f'{source_dir}/messages.pot', ignore_context=True
+    )
+    target_contents = get_template_content(
+        f'{target_dir}/messages.pot', ignore_context=True
+    )
+
+    success = source_contents == target_contents
+
+    print_filename_results(f'{source_dir}/messages.pot', success)
+
+    return success
+
+
+def build_schema_templates(output_dir):
+    docker_client = docker.from_env()
+
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f'{schema_name}.pot'
+        schema_file = f'{schema_name}.json'
+
+        docker_client.images.pull('onsdigital/eq-translations:latest')
+        run_results = docker_client.containers.run(
+            'onsdigital/eq-translations',
+            f'pipenv run python -m cli.template_extractor ../eq-survey-runner/data/en/{schema_file} /usr/src/templates',
+            volumes={
+                os.getcwd(): {'bind': '/usr/src/eq-survey-runner', 'mode': 'rw'},
+                output_dir: {'bind': '/usr/src/templates', 'mode': 'rw'},
+            },
+            remove=True,
+        )
+        logger.info('Building %s/%s', output_dir, template_file)
+        logger.info(run_results.decode())
+
+
+def check_schema_templates(source_dir, target_dir):
+    success = True
+
+    for schema_name in SCHEMAS_TO_EXTRACT:
+        template_file = f'{schema_name}.pot'
+
+        source_contents = get_template_content(f'{source_dir}/{template_file}')
+        target_contents = get_template_content(f'{target_dir}/{template_file}')
+        template_the_same = source_contents == target_contents
+
+        if not template_the_same:
+            success = False
+
+        print_filename_results(f'app/translations/{template_file}', template_the_same)
+
+    return success
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Extract translation templates from runner'
+    )
+    parser.add_argument(
+        '--check', help='Test the templates without making changes', action='store_true'
+    )
+
+    args = parser.parse_args()
+
+    if args.check:
+        logger.info('--- Testing Translations ---')
+
+        with tempfile.TemporaryDirectory(dir='/tmp') as temp_dir:
+            build_static_template(f'{temp_dir}/messages.pot')
+            build_schema_templates(temp_dir)
+
+            static_success = check_static_template('app/translations', temp_dir)
+            dynamic_success = check_schema_templates('app/translations', temp_dir)
+
+            if not all((dynamic_success, static_success)):
+                logger.error(
+                    'Changes to translation source strings have been found. Please regenerate templates.'
+                )
+                sys.exit(1)
+
+            logger.debug('No changes to translation source strings have been found.')
```

Comment:
I'm not sure that 'changes' is the right language here. I think it's more about checking that the template files are up to date. It would also be helpful to print the command to fix the problem. So we could say:

"Translation templates are not up to date. Run `make translation-templates` to fix this" 

or 

"Translation templates are up to date"